### DISCUSSION TITLE
CSLa Samples RazorPagesExample

### DISCUSSION THREAD

**Discussion by Art666OTS** [MarimerLLC/csla#2926] [Questions] [answered]
Hi All,

I tried to run the sample RazorPagesExample and in VS2022 and I get a compile error.
![razorExampleError](https://user-images.githubusercontent.com/38965119/167533125-28475334-44a9-4292-8108-05dddd980e01.PNG)

Does anyone have a suggestion for how to correct this?

**Comment by rockfordlhotka** [comment, marked as answer]
Is this the one from `main`? I'll be updating all the samples to use the release version of CSLA 6 in the next few days.
